app/src/main/java/org/sopt/and/dto/Auth.kt

```diff
@@ -0,0 +1,48 @@
+package org.sopt.and.dto
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+/* 유저 등록 */
+@Serializable
+data class RequestUserRegistrationData(
+    @SerialName("username")
+    val userName: String,
+    @SerialName("password")
+    val password: String,
+    @SerialName("hobby")
+    val hobby: String
+)
+
+@Serializable
+data class ResponseUserRegistration(
+    @SerialName("result")
+    val result: ResultUserNo
+)
+
+@Serializable
+data class ResultUserNo(
+    @SerialName("no")
+    val no: Int
+)
```

Comment:
DTO같은 경우는 이름에 Dto라고 명시해주는 것이 좋습니다
또한 Response, Request등을 보통은 다른 파일로 빼는 편입니다.

```suggestion
@Serializable
data class RequestUserRegistrationDto(
    @SerialName("username")
    val userName: String,
    @SerialName("password")
    val password: String,
    @SerialName("hobby")
    val hobby: String
)

@Serializable
data class ResponseUserRegistrationDto(
    @SerialName("result")
    val result: ResultUserNo
)

@Serializable
data class ResponseUserRegistrationDto(
    @SerialName("no")
    val no: Int
)
```

이런식으로요! 각각 Dto는 다른 파일에 넣어주시는게 조금 더 좋을 것 같습니다.
그렇게 하면 주석으로 해당 Dto들을 나눠주는 것도 필요 없겠죠?